PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt

```diff
@@ -126,6 +126,10 @@ class PayPalCheckoutRequest @JvmOverloads constructor(
 
         userPhoneNumber?.let { parameters.put(PHONE_NUMBER_KEY, it.toJson()) }
 
+        shopperSessionId?.let {
+            if (it.isNotEmpty()) parameters.put(SHOPPER_SESSION_ID, it)
+        }
```

Comment:
We should be able to use the same simplified putOpt logic from the `PayPalVaultRequest` here